firmware/spiders/netgear.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import re
+from typing import Generator
+
+from scrapy import Request
+from scrapy.http import Response
+
+from firmware.custom_spiders import FirmwareSpider
+from firmware.items import FirmwareItem
+
+
+class Netgear(FirmwareSpider):
+    name = 'netgear'
+    manufacturer = 'NETGEAR'
+
+    start_urls = [
+        'https://www.netgear.com/de/home/wifi/routers/',  # router
+        'https://www.netgear.com/de/home/wifi/range-extenders/',  # repeater
+        'https://www.netgear.com/de/home/wifi/mesh/',  # mesh
+        'https://www.netgear.com/de/home/online-gaming/routers/',  # 'gaming' router
+        'https://www.netgear.com/de/home/mobile-wifi/hotspots/'  # 4g/5g router
+        'https://www.netgear.com/de/home/mobile-wifi/lte-modems/',  # 4g/5g router
+    ]
+
+    custom_settings = {
+        # Sorry.
+        'ROBOTSTXT_OBEY': False,
+        # being nice to Netgear servers
+        'CONCURRENT_REQUESTS': 1,
+        'CONCURRENT_ITEMS': 1,
+        'DOWNLOAD_DELAY': 0.75,
+        'RANDOMIZE_DOWNLOAD_DELAY': True,
+        'REFERER_ENABLED': False
```

Comment:
Yeah, thought about that as well, we could set them as global scrapy settings. I hesitated to do so because I thought it would be nice to explicitly state behavioral features for each one of them.

For the time being, I'm going to leave it 'as is' and may revise later down the road.